site/topics/control-logic/add_subtract_output.hex

```diff
@@ -0,0 +1,17 @@
+v2.0 raw
```

Comment:
this one is
0 --- Load into A contents of address 14
1 --- Load into B contents of address 15
2 --- Add B to A and store result in A
3 --- Save A to address 12
4 --- Output unsigned int from address 12
5 --- Load into A contents of address 14
6 --- Load into B contents of address 15 (not actually necessary)
7 --- Subtract B from A and store in A
8 --- Save A to address 13
9 --- Output signed int from address 13
10 --- halt
11
12
13
14 --- 31
15 --- 32